### PR TITLE
Add Memory Value Humanization docs

### LINKED ISSUE #7539: Document humanize memory quantities feature
**Which component are you using?**:
VPA

**Is your feature request designed to solve a problem? If so describe the problem this feature should solve.**:

**Describe the solution you'd like.**:
A new feature was introduced in [PR #7428](https://github.com/kubernetes/autoscaler/pull/7428), but documentation for this feature is currently missing.
Reference: [Comment on the PR](https://github.com/kubernetes/autoscaler/pull/7428#issuecomment-2436137446).
As the author of this feature, I am happy to take on the task of documenting it.

**Describe any alternative solutions you've considered.**:
NONE

**Additional context.**:
NONE

### PR DESCRIPTION
**Pull request by omerap12**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
Add docs for humanize-memory feature
#### Which issue(s) this PR fixes:

Fixes #7539

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

File: vertical-pod-autoscaler/docs/features.md [hunk 1]
```diff
@@ -16,3 +17,18 @@ with VPA resource policy, VPA will follow VPA policy (and set values outside the
...
+When enabled, memory values in recommendations will be:
+- Converted to the most appropriate binary unit (KiB, MiB, GiB, or TiB)
+- Displayed with up to 2 decimal places for precision
+- Applied to target, lower bound, and upper bound recommendations
+
+For example, instead of seeing a memory recommendation of `262144000` bytes, you would see `250.00Mi`.
+
+To enable this feature, set the `--humanize-memory` flag to true when running the VPA recommender:
+```bash
+--humanize-memory=true
```
Comment (adrianmoisey) [inline vertical-pod-autoscaler/docs/features.md:34]: there is a missing ``` at the end of this file

Comment (omerap12) [reply vertical-pod-autoscaler/docs/features.md:34]: Thanks, fixed!

File: vertical-pod-autoscaler/docs/features.md [hunk 2]
```diff
@@ -16,3 +17,19 @@ with VPA resource policy, VPA will follow VPA policy (and set values outside the
...
 
 To disable getting VPA recommendations for an individual container, set `mode` to `"Off"` in `containerPolicies`.
+
+## Memory Value Humanization
+
+VPA can present memory recommendations in human-readable binary units (KiB, MiB, GiB, TiB) instead of raw bytes, making resource recommendations easier to understand. This feature is controlled by the `--humanize-memory` flag in the recommender component.
+
+When enabled, memory values in recommendations will be:
+- Converted to the most appropriate binary unit (KiB, MiB, GiB, or TiB)
+- Displayed with up to 2 decimal places for precision
```
Comment (adrianmoisey) [inline vertical-pod-autoscaler/docs/features.md:27]: Is it worth mentioning that this feature effectively rounds the recommendation up a little, meaning that the recommendation could be artificially bloated a little?

Comment (omerap12) [reply vertical-pod-autoscaler/docs/features.md:27]: Added

**Comment by adrianmoisey** [comment]
/lgtm

Yay for documentation!

**Comment by raywainman** [comment]
:) Thanks for the docs!

/lgtm

/approve

**Pull request merged**